app/views/devise/registrations/edit.html.haml

```diff
@@ -66,6 +66,22 @@
       %p.mt-1.p Danger zone! Once you delete you account, there is no going back.
 
     .card.md:col-span-2
-      .card-body
+      .card-body{"x-data" => "{ button: true, open: false }"}
         .space-y-6
-          = button_to "Delete Account", registration_path(resource_name), {data: { turbo_confirm: "Are you sure?" }, class: 'text-danger text-sm mr-6', method: :delete}
+          %div{ "x-show" => "button" }
+            .text-sm.text-neutral-500.mb-2 Delete your account. This action cannot be undone and all your data will be deleted from our system.
+            %button{ class: 'text-danger text-sm mr-6', method: :delete, "x-on:click" => "open = ! open; button = ! button" } Delete Account
+        = form_for(resource, as: resource_name, url: registration_path(resource_name), builder: RapidRailsFormBuilder, html: { method: :delete }) do |f|
+          .text-black{"x-show" => "open"}
```

Comment:
don't do text-black or any colors like this. we have base styles. you should just use `%p.p` (p also being the class). This handles dark mode, etc. 

The current code does not work for dark mode. 

<img width="1167" alt="image" src="https://github.com/danielpaul/RapidRails/assets/333233/23847fb2-b224-4fad-a27b-d75bf4ac4360">